plasma_framework/contracts/mocks/utils/Imports.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity ^0.5.0;
```

Comment:
haha I like this `Imports.sol` file. Probably we can just put it directly under `mocks/` as it would be used as a global importing file not only for contract/tests in `utils/`? how do you think?

Reply:
Yep, makes sense 👍